t/node/upstream.t

```diff
@@ -26,7 +26,69 @@ run_tests();
 
 __DATA__
 
-=== TEST 1: set upstream(id: 1)
+=== TEST 1: set upstream(id: 1) invalid parameters
```

Comment:
It is best to add new test cases to the end of the file. This will not affect any existing test cases.
@nic-chen